cliquet_docs/reference/notifications.rst

```diff
@@ -3,30 +3,64 @@
 Notifications
 #############
 
-Knowing when a bucket or a collection has been modified is very
+Knowing collection has been modified is very
 useful to integrate a Cliquet-based application with other services.
 
 For example, a search service that gets notified everytime something
 has changed, can continuously update its indexes.
 
 Cliquet leverages Pyramid's built-in event system and produces
-a :class:`cliquet.events.ResourceChanged` event everytime a resource has
-been modified. Event listeners can then pick up those events and act upon them.
+a :class:`cliquet.events.ResourceChanged` event everytime a record in a
+:ref:`resource` has been modified.
 
-The :class:`cliquet.events.ResourceChanged` event contains a payload with
+Event listeners can then pick up those events and act upon them.
+
+.. code-block:: python
+
+    from cliquet.events import ResourceChanged
+
+
+    def on_resource_changed(event):
+        resource_name = event.payload['resource_name']
+        action = event.payload['action']
+
+        if resource_name == 'user' and action == 'create':
+            send_welcome_email(event.payload['user_id'])
+
+    config.add_subscriber(on_resource_changed, ResourceChanged)
+
+
+The :class:`cliquet.events.ResourceChanged` event contains a ``payload`` attribute with
 the following information:
 
 - **timestamp**: the time of the event
 - **action**: what happened. 'create', 'update' or 'delete'
 - **uri**: the uri of the impacted resource
 - **user_id**: the authenticated user id
-- **resource_name**: the name of the impacted resouce. 'bucket', 'collection'
-  or 'record'
-- **<resource_name>_id**: id of the impacted resouce
+- **resource_name**: the name of the impacted resouce (e.g. 'mushroom', 'bucket',
+  etc.)
+- **<resource_name>_id**: id of the impacted record
+- **<match-dict value>**: every matchdict values of the URL pattern
+
+
+Event listeners
+---------------
+
+It is possible for an application or a plugin to listen to the events and execute
+some code. In this case, the notifications are synchronous.
+
+In addition to this low-level events handling, *Cliquet* provides some generic
```

Comment:
> Does low-level mean Pyramid listeners ?

Hum, yes.

What I mean : 

- *Cliquet* is a toolkit. The developer of an app (or a plugin) can subscribe to the event manually using Pyramid `config.add_event_subscriber()`
- *Cliquet* has pluggable abstractions. The Cliquet-based application can be configured so that events are streamed to Redis.

> Do we have Pyramid listeners configured in Kinto ?

Kinto does not exist here :)